_commands/tools/asciinema.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+asciinema
+---
+`aciinema` is a tool that allows you to record a terminal session with out having to use screen capture. It also allows for easy copy-pasting!
+<!-- one line explanation would go here -->
+
+<!-- minimal example -->
+~~~ bash
+$ sudo apt-get install asciinema #install
+$ asciinema rec #start session
+$ your_commands_here
+$ exit #exit session
+~~~
+
+<!--more-->
+
+### Examples
+
```

Comment:
Could you include a list of possible commands (e.g. rec and exit) in a list, like had been done for [asciiquarium](https://c4cs.github.io/commands/asciiquarium)?